mod/src/main/java/gg/skytils/skytilsmod/mixins/transformers/network/MixinNetHandlerPlayClient.java

```diff
@@ -52,4 +62,57 @@ private void onHandleSpawnMobTail(
             MasterMode7Features.INSTANCE.onMobSpawned(entity);
         }
     }
+
+    //# if MC>=10900
+    private static final MinecraftClient client = MinecraftClient.getInstance();
+
+    @Inject(method = "onBlockUpdate", at = @At("TAIL"))
+    private void onBlockUpdate(BlockUpdateS2CPacket packet, CallbackInfo ci) {
+        if (client.world == null) return;
+        if (!Skytils.getConfig().getGlassPaneDesync()) return;
+
+        for (Direction dir : Direction.Type.HORIZONTAL) {
+            BlockPos neighborPos = packet.getPos().offset(dir);
+            BlockState neighborState = client.world.getBlockState(neighborPos);
+            if (neighborState.getBlock() instanceof PaneBlock) {
+                client.world.setBlockState(neighborPos, updateState(neighborPos, neighborState));
+            }
+        }
+    }
+
+    private BlockState updateState(BlockPos pos, BlockState state){
```

Comment:
```suggestion
    @Unique
    private BlockState updateState(BlockPos pos, BlockState state){
```
This will also require an import